Source/WebDriver/Session.h

```diff
@@ -275,12 +276,16 @@ public RefCounted<Session>
     HashSet<String> m_globalEventSet;
     WeakPtr<WebSocketServer> m_bidiServer;
 
+    // Relayed messages
```

Comment:
Naming nit: `m_pendingCommands` or `m_pendingRelayedCommands`

Reply:
I actually removed this, as we don't need to keep track of the completion handler given the `WebSocketServer` maps the connection to a given session.